chi/server.py

```diff
@@ -178,7 +180,9 @@ def __init__(self, id=None, lease=None, key=None, image=DEFAULT_IMAGE,
                 net_ids=net_ids,
                 **kwargs
             )
-            self.server = self.nova.servers.create(**server_kwargs)
+            self.server = self.conn.compute._get_resource(
```

Comment:
Does `_get_resource` create the resource?

Reply:
I think you may need

```python
self.server = self.conn.compute.create_server(**server_kwargs)
```

Reply:
It did create the resource, but switching to `create_server` anyway.